compiler-cli/src/main.rs

```diff
@@ -207,6 +208,13 @@ enum Command {
         dev: bool,
     },
 
+    /// Remove project dependencies
+    Remove {
+        /// The names of Hex packages to remove
```

Comment:
```suggestion
        /// The names of packages to remove
```

This will work for non-Hex packages also